.github/ISSUE_TEMPLATE/feature_request.md

```diff
@@ -0,0 +1,17 @@
+---
+name: Feature request
+about: Suggest an idea for this project
+title: ''
+labels: enhancement
+assignees: ''
+
+---
+
+[//]: #  (Before raising the feature request, please check to see if an existing feature request already exists.)
+
+
+**Is your feature request related to a problem? Please describe.**
```

Comment:
Would some more structure around what doesn't work, why it would be helpful, and how it might be done be useful?

For example: https://github.com/aws/aws-cdk/blob/master/.github/ISSUE_TEMPLATE/feature-request.yml